src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_pubsub.v1.json.baseline

```diff
@@ -18,7 +18,7 @@ authorize(function(authClient) {
   var request = {
     // REQUIRED: The resource for which the policy is being requested. `resource` is usually specified as
     // a path. For example, a Project resource is specified as `projects/{project}`.
-    resource: '',  // TODO: Update placeholder value.
+    resource_: '',  // TODO: Update placeholder value.
```

Comment:
it does:
https://github.com/google/google-api-nodejs-client/blob/master/apis/pubsub/v1.js#L1762